resources/js/components/dues/SelfServiceAccessOverride.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="col-sm-6 com-md-3 col-lg-4"
+       v-if="!loading && eligibility && (eligibility.eligible || eligibility.user_rectifiable)">
```

Comment:
Idk how I feel about the card showing up for membership agreements. There's already a (hopefully very clear) card for that.

Care amount=4

Reply:
The reason to show both is because while the self-service override doesn't appear, it's not apparent what has to be done to receive one

Reply:
I feel like "Start Here" is pretty apparent, but I'm not going to push on this.